code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -186,13 +186,15 @@
 	..()
 
 /mob/living/carbon/human/proc/Stasis(amount)
-	if((species.flags & NO_SCAN) || isSynthetic())
+	//if((species.flags & NO_SCAN) || isSynthetic()) vorestation removal
+	if(((species.flags & NO_SCAN) && !istype(species, /datum/species/xenochimera)) || isSynthetic()) //vorestation addition

```

Comment:
Strictly speaking, the things that cause cloneloss are things that should fuck them up. Like, a lot. I was originally going to have it (along with brain) be one of the things that regenning didn't clear properly and actually need actual treatment, along with having their cloneloss levels as a thing that increases revival time. Shooting them with a decloner makes sense as a thing that would _ruin_ them.